compiler/src/iree/compiler/Pipelines/Options.h

```diff
@@ -220,6 +222,12 @@ struct DispatchCreationOptions {
   bool enableAggressiveFusion = false;
   bool enableFuseMultiUse = true;
 
+  // Enables data tiling in dispatch creation phase. There are two data-tiling
+  // flags during the transition state. The other has to be off if it is
```

Comment:
```suggestion
  // flags during the transition state. The other has to be off if this one is
```
For clarity, this is what you meant right?